Description: Disable doctests
 Disable doctests since they fail on too many architectures, with what seems to
 be a GHC bug.

Index: b/rank2classes.cabal
===================================================================
--- a/rank2classes.cabal
+++ b/rank2classes.cabal
@@ -49,6 +49,7 @@ library
 
 test-suite doctests
   type:                exitcode-stdio-1.0
+  buildable: false
   hs-source-dirs:      test
   default-language:    Haskell2010
   main-is:             Doctest.hs
